packages/core/src/modules/vc/SignatureSuiteRegistry.ts

```diff
@@ -11,7 +11,7 @@ export interface SuiteInfo {
   suiteClass: typeof LinkedDataSignature
   proofType: string
   requiredKeyType: string
-  keyType: string
+  keyTypes: [KeyType]
```

Comment:
This should be `KeyType[]`. This actually means an array with one item of type `KeyType` (you can use this to create tuple types)

```suggestion
  keyTypes: KeyType[]
```

Reply:
Hmm after thinking about it a bit more. This is the key type as we use it in the framework, but not the key as is used in did documents. Those key types are e.g. Ed25519Key2018 (or something like it). So we still have to make the transformation, but only the signature suites know which actual types is supports. 

Should this be updated to include the supported key types as used by did documents? E.g. if the suites only supports 2018 and 2020 ed25519 keys and there's a new 2022 one that isn't supported we should only extract the 2018 and 2022 keys from the did document, as that's the only ones we can sign with.